### DISCUSSION TITLE
how to check matcher in rule order

### DISCUSSION THREAD

**Discussion by passwa11** [projectdiscovery/nuclei-templates#9681] [Q&A]
body: bbbb ccccc aaaa

contains_all(body, 'aaaa', 'bbbb', 'cccc')

false matchers now will search all strings?

**Comment by GeorginaReeder** [comment]
Hi there @passwa11 ! I can see you've already posted this question here https://github.com/projectdiscovery/nuclei/discussions/5123

Closing this thread in favour of the previous one. Our team will take a look as soon as they can!
